src/plugins/intel_cpu/src/nodes/strided_slice.cpp

```diff
@@ -222,6 +272,7 @@ void StridedSlice::initSupportedPrimitiveDescriptors() {
         config.inConfs[AXES_ID].constant(isConstantInput[AXES_ID]);
     }
     config.outConfs.resize(1);
+    config.outConfs[0].inPlace(isOptimizedOut ? 0 : -1);
```

Comment:
Also it makes sense to assign "unknown" implementation type to raise the priority and distinguish the implementation from the default one.